server/lib/authorize.js

```diff
@@ -0,0 +1,40 @@
+const axios = require('axios');
+
+const querystring = require('querystring');
```

Comment:
https://nodejs.org/api/querystring.html#querystring_query_string
The native `querystring` module is considered legacy - and I'd argue it's kind of unnecessary. Maybe just assemble it manually?